src/cmds/net/telnetd/telnetd.c

```diff
@@ -23,6 +23,7 @@
 
 #include <util/math.h>
 
+#include <kernel/printk.h>
```

Comment:
Where `printk` is required?

Reply:
In 173 row:
`printk("system return error: %d\n", ret);`

Reply:
Ah, you just moved it to another place, now I see